app/components/total-monthly-estimate/component.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+
+const HOURS_PER_MONTH = 730;
```

Comment:
This is already specified in https://github.com/aptible/dashboard.aptible.com/blob/c60a1ac7981d166bd4df2493ebf5ea13faab0a23/app/components/usage-quote-by-resource/component.js#L3

Can we have it in one location and import from there?